README.md

```diff
@@ -1,40 +1,55 @@
 
-Azure Sentinel is a SIEM (Security Event Information Management) and a SOAR Security Orchestrated Response) that Microsoft has created. Azure Sentinel  delivers security and threat intelligence and proving a single solution for alert detection, threat visibility, proactive hunting and threat response
+Azure Sentinel is both a SIEM (Security Information and Event Management) and a SOAR (Security Orchestration, Automation, and Response) solution that Microsoft has created. Azure Sentinel delivers security and threat intelligence and proving a single solution for alert detection, threat visibility, proactive hunting, and threat response.
 
-## Data Connectors 
-With Azure Sentinel you can connect all your data sources with data connectors. Sentinel comes with multiple built in connections for Microsoft solutions. For example, you can connect Office 365 for real time integration
+## Data Connectors
+With Azure Sentinel, you can connect all your data sources with data connectors. Sentinel comes with multiple built-in connections for Microsoft solutions. For example, you can connect Office 365 for real time integration.
+
+## Workbooks
+Once you connect your resources to Azure Sentinel, you can monitor the data. You can use the built-in Workbooks that Azure provides or you can make custom workbooks. You can build dashboards from the workbooks to visualize the data within the Azure Portal.
 
-## Workbooks 
-Once you connect you resources to Azure Sentinel you can monitor the data. You can use the built in Workbooks that Azure provides or you can make custom workbooks. You can build dashboards from the workbooks to visualize the data within the Azure Portal. 
 ![](images/workbooks.PNG)
+
 ## Analytics
-Analytics are rules that can help minimize that amount of threats that have to be reviewed and investigated. You can use Analytics to create Incidents. Analytics ranks severity from high to medium to low to informational. Azure Sentinel will automatically rank the rules them from most sever to the least severe. Below is an example of what Analytics can look in aby companies workspace
+Analytics are rules that can help minimize that amount of threats that have to be reviewed and investigated. You can use Analytics to create Incidents. Analytics ranks severity from high to medium to low to informational. Azure Sentinel will automatically rank the rules from most severe to the least severe. Below is an example of what Analytics can look like in any company's workspace.
+
 ![](images/analytics.PNG)
-## Incidents 
-Incidents are a group of related alerts that are created to investigate an solve any actionable threat.Like Analytics, incidents are also ranked by severity 
+
+## Incidents
+Incidents are a group of related alerts that are created to investigate an solve any actionable threat. Like Analytics, incidents are also ranked by severity.
```

Comment:
I'm not sure how Sentinel uses the terms, but alerts are _potential_ problems and incidents are legit bad. Make sure you're being clear or people familiar with these general terms will be confused.